app/views/setup/welcome.html.slim

```diff
@@ -3,8 +3,8 @@
   button.close[type="button" data-dismiss="alert" aria-label="Close"]
     span[aria-hidden="true"]
       | &times;
-  i.fa.fa-eye.fa-4x.pull-left aria-hidden="true"
-  span Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span In order to secure transactions and communication between your cluster nodes, it is required to generate certificates. Please enter the details for your certificate authority (CA).
```

Comment:
I'm not sure if it would be good to add something like: `[Learn more](https://kubernetes.io/docs/admin/authentication/)`

Maybe we can add a link to a documentation page in the future. Feel free to skip it for now.